tests/test_tensorflow_data_api.py

```diff
@@ -0,0 +1,161 @@
+"""Tests for TileDB integration with Tensorflow Data API."""
+
+import os
+import tiledb
+import numpy as np
+
+import tensorflow as tf
+from tensorflow.keras.models import Sequential
+from tensorflow.keras.layers import (
+    Dense,
+    Flatten,
+    Dropout,
+)
+from tensorflow.python.keras import testing_utils
+from tensorflow.python.platform import test
+
+from tiledb.ml.data_apis.tensorflow import TensorflowTileDBDataset
+
+# Suppress all Tensorflow messages
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+
+
+def ingest_in_tiledb(data: np.array, batch_size: int, uri: str):
+    # First dimension
+    dims = [
+        tiledb.Dim(
+            name="dim_1", domain=(0, data.shape[0] - 1), tile=batch_size, dtype=np.int32
+        )
+    ]
+
+    # Remaining dimensions
+    for dim in range(1, data.ndim):
+        dims.append(
+            tiledb.Dim(
+                name="dim" + str(dim),
+                domain=(0, data.shape[dim] - 1),
+                tile=data.shape[dim],
+                dtype=np.int32,
+            )
+        )
+
+    # TileDB schema
+    schema = tiledb.ArraySchema(
+        domain=tiledb.Domain(dims),
```

Comment:
Does this work? I thought `Domain` takes variable args (`tiledb.Domain(*dims)`) not a list.

Reply:
Yeap, it works. But I will change it ;)